cmd/containerd-shim-runhcs-v1/log.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Not sure on if we need an example logging binary included (this is probably just so reviewers can get a feel for the work I assume so correct me if I'm wrong). Assuming we keep this, I think a better place for the binary would just be in the /cmd/logging package you've made already.

Reply:
I'll remove this and update pr description with the link to an upstream example